pyaedt/icepak.py

```diff
@@ -4869,3 +4869,341 @@ def assign_symmetry_wall(
                 raise SystemExit
         except (GrpcApiError, SystemExit):
             return None
+
+    @pyaedt_function_handler()
+    def assign_recirculation_opening(self, face_list, extract_face, thermal_specification="Temperature",
+                                     assignment_value="0cel", conductance_external_temperature=None,
+                                     flow_specification="Mass Flow", flow_assignment="0kg_per_s_m2",
+                                     flow_direction=None, start_time=None, end_time=None, boundary_name=None):
+        """Assign recirculation faces.
+
+        Parameters
+        ----------
+        face_list : list
+            List of modeler.cad.elements3d.FacePrimitive or of integers
+            containing faces ids.
+        extract_face : modeler.cad.elements3d.FacePrimitive, int
+             Face of the face on the extract side.
+        thermal_specification : str, optional
+            Type of the thermal assignment across the two recirculation
+            faces. Options are ``"Conductance"``, ``"Heat Input"`` and
+            ``"Temperature"``. Default is ``"Temperature"``.
+        assignment_value : str or dict, optional
+            String with value and units of the thermal assignment. For a
+            transient assignment, a dictionary can be used. The dictionary
+            should contain two keys: ``"Function"`` and ``"Values"``.
+            - For the ``"Function"`` key, acceptable values are
+            ``"Exponential"``, ``"Linear"``, ``"Piecewise Linear"``,
+            ``"Power Law"``, ``"Sinusoidal"``, and ``"Square Wave"``.
+            - For the ``"Values"`` key, a list of strings containing the
+            parameters required by the ``"Function"`` key selection. For
+            example, when``"Linear"`` is set as the ``"Function"`` key, two
+            parameters are required: the value of the variable at t=0 and the
+            slope of the line. For the parameters required by each
+            ``"Function"`` key selection, see the Icepak documentation.
+            The parameters must contain the units where needed.
+            The default value is ``"0cel"``.
+        conductance_external_temperature : str, optional
+            External temperature value, needed if ``thermal_specification``
+            is set to ``"Conductance"``. Default is ``None``.
+        flow_specification: str, optional
```

Comment:
```suggestion
        face_list : list
            List of ``modeler.cad.elements3d.FacePrimitive`` objects or a list of integers
            containing faces IDs.
        extract_face : modeler.cad.elements3d.FacePrimitive, int
             Face of the face on the extract side.
        thermal_specification : str, optional
            Type of the thermal assignment across the two recirculation
            faces. The default is ``"Temperature"``. Options are ``"Conductance"``,
            ``"Heat Input"``, and ``"Temperature"``.
        assignment_value : str or dict, optional
            String with value and units of the thermal assignment. For a
            transient assignment, a dictionary can be used. The dictionary
            should contain two keys: ``"Function"`` and ``"Values"``.

            - For the ``"Function"`` key, acceptable values are
            ``"Exponential"``, ``"Linear"``, ``"Piecewise Linear"``,
            ``"Power Law"``, ``"Sinusoidal"``, and ``"Square Wave"``.
            - For the ``"Values"`` key, a list of strings containing the
            parameters required by the ``"Function"`` key selection. For
            example, when``"Linear"`` is set as the ``"Function"`` key, two
            parameters are required: the value of the variable at t=0 and the
            slope of the line. For the parameters required by each ``"Function"``
            key selection, see the Icepak documentation.
            The parameters must contain the units where needed.
            The default value is ``"0cel"``.
        conductance_external_temperature : str, optional
            External temperature value, which is needed if ``thermal_specification``
            is set to ``"Conductance"``.  The default is ``None``.
        flow_specification: str, optional
```
For the first parameter, can we rewrite "List of ``modeler.cad.elements3d.FacePrimitive`` objects to plain English? Also, for the second parameter, is "Face of the face" correct? Just checking!

Reply:
It is possible to use both a fixed value or a dictionary for transient assignment. By default a fixed value is used.